From b217d7591772d729dcc002060663b63c62c2f1ce Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jo=C3=A3o=20Batista?= <joao42lbatista@gmail.com>
Date: Sun, 26 May 2024 19:36:40 -0300
Subject: [PATCH] meson.build - v0.7.2

---
 meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meson.build b/meson.build
index 76a0340..c624622 100644
--- a/meson.build
+++ b/meson.build
@@ -1,6 +1,6 @@
 project('MangoHud',
   ['c', 'cpp'],
-  version : 'v0.7.1',
+  version : 'v0.7.2',
   license : 'MIT',
   meson_version: '>=0.60.0',
   default_options : ['buildtype=release', 'c_std=c99', 'cpp_std=c++14', 'warning_level=2']
-- 
2.45.1

