From 61fb38b8da3f5f21e4d0891334d966fcd0de43ac Mon Sep 17 00:00:00 2001
From: FlightlessMango <flightlessmangoyt@gmail.com>
Date: Wed, 14 Feb 2024 01:08:37 +0100
Subject: [PATCH] reset_fps_metrics: check that metrics is inited first

---
 src/keybinds.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/keybinds.cpp b/src/keybinds.cpp
index f328e05..395eb55 100644
--- a/src/keybinds.cpp
+++ b/src/keybinds.cpp
@@ -99,6 +99,7 @@ void check_keybinds(struct overlay_params& params, uint32_t vendorID){
    if (elapsedF12 >= keyPressDelay &&
        keys_are_pressed(params.reset_fps_metrics)) {
       last_f12_press = now;
-      fpsmetrics->reset_metrics();
+      if (fpsmetrics)
+         fpsmetrics->reset_metrics();
    }
 }
-- 
2.44.0

